app/views/delayed/web/jobs/index.html.erb

```diff
@@ -7,13 +7,13 @@
 <table class="table table-bordered">
   <thead>
     <tr>
-      <th>ID</th>
-      <th>Status</th>
-      <th>Created At</th>
-      <th>Run At</th>
-      <th>Attempts</th>
-      <th style="width: 190px;">Last Error</th>
-      <th style="width: 115px;">Actions</th>
+      <th><%= t('delayed/web.views.id') %></th>
+      <th><%= t('delayed/web.views.status') %></th>
+      <th><%= t('delayed/web.views.created_at') %></th>
+      <th><%= t('delayed/web.views.run_at') %></th>
+      <th><%= t('delayed/web.views.attempts') %></th>
+      <th style="width: 190px;"><%= t('delayed/web.views.last_error') %></th>
+      <th style="width: <%= I18n.locale == :en ? '115' : '160' %>px;"><%= t('delayed/web.views.actions') %></th>
```

Comment:
Sure, but what happens if someone adds another language and then the optimal width is 130px. The point is that we should pick the route that has the least amount of effort to maintain over time. Consider picking a middle size and then using CSS to truncate. Eg: `text-overflow`.